app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoParser.kt

```diff
@@ -21,9 +21,24 @@ object PocketVideoParser {
             url = jsonObj.optString("tv_url", null) ?: jsonObj.getString("url"),
             thumbnailURL = jsonObj.getString("image_src"),
             popularitySortId = jsonObj.getInt("popularity_sort_id"),
-            authors = jsonObj.getString("authors")
+            authors = getAuthorName(jsonObj)
         )
     } catch (e: JSONException) {
         null
     }
+
+    private fun getAuthorName(jsonObj: JSONObject): String {
+        var authorName = ""
+        try {
+            val authorsJSON = JSONObject(jsonObj.getString("authors"))
+
+            for (x in authorsJSON.keys()) {
+                val authorObject = JSONObject(authorsJSON[x].toString())
+                authorName = authorObject.get("name").toString()
+            }
```

Comment:
I had forgotten about this, but the requirements for this story specify our fallback behavior.  If 'author' is not present, we should show "YouTube".  So we should use that as a default instead of an empty string.

We can break DEFAULT_AUTHOR out into a file level constant.

Reply:
Good point. Although, I don't think we can always show YouTube since some videos are coming from other places, like BBC. The platform would have to be determined based on the video, as in previous functionality.
I'll add that back as the fallback behaviour.

Reply:
Is there a way to add this behaviour in the parser? I can't figure it out since I need the url field of the video and context to pass to `FormattedDomain#format`. Is it okay to leave it in the fragment?

Reply:
Formatting the domain doesn't seem like it should be a responsibility of the parser, so we don't need to pass it in there.  It looks like it's currently being done in the RecyclerView.Adapter, which isn't great either.  If it's out of scope for this change we can put a TODO to fix it, but this is a transformation that should really be handled by the view model, imo.